packages/components/src/GeometryPanel.component.svelte

```diff
@@ -0,0 +1,209 @@
+<svelte:options customElement="geometry-panel" />
+
+<script lang="typescript">
+  import { onMount, onDestroy } from 'svelte';
+  import Section from './Section.component.svelte';
+  import TypedTextBox from './TypedTextBox.component.svelte';
+  import { Events } from './constants/events';
+  import { ButtonProps, GUIEvent } from './types';
+
+  export let attributes = '[]';
+
+  let selectedGeometry = undefined;
+
+  function loadPickedObject(event: CustomEvent<GUIEvent>) {
+    const target = event.detail.target.split('_');
+
+    if (target[0] !== 'geometry') return;

```

Comment:
I'd prefer to have everywhere same styling for `if` statements. Even though you save one line
```javascript
if (target[0] !== 'geometry') {
  return;
}
```